src/modules/dashboard/components/ColonyActions/ColonyActions.tsx

```diff
@@ -31,6 +31,7 @@ import {
 } from '~types/index';
 
 import styles from './ColonyActions.css';
+import { createAddress } from '~utils/web3';
```

Comment:
A bit of a nitpick but this import should be grouped with the other relative imports

Reply:
Good pickup, a symptom of using auto-complete on function calls.